compiler/optimizer/OMRTransformUtil.cpp

```diff
@@ -386,3 +386,32 @@ OMR::TransformUtil::removeTree(TR::Compilation *comp, TR::TreeTop * tt)
    {
    comp->getJittedMethodSymbol()->removeTree(tt);
    }
+
+/**
+ * \brief
+ *    Transform a call node to a TR::PassThrough node with one child as a safe way to eliminate the call.
+ *
+ * \parm opt
+ *    The optimization asking for this transformation.
+ *
+ * \parm node
+ *    The call node to be transformed.
+ *
+ * \parm anchorTree
+ *    The tree before which the children of the call node are to be anchored.
+ *
+ * \parm child
+ *    The child for node after the transformation. It's the receiver of the call or any node for a static call.
+ *
+ * \note
+ *    The call node's refcount has to be 1 since the transformation is equivalent to eliminating the call,
+ *    otherwise the resulting node will cause problems in other optimizations.
+ */
+void
+OMR::TransformUtil::transformCallNodeToPassThrough(TR::Optimization* opt, TR::Node* node, TR::TreeTop * anchorTree, TR::Node* child)
```

Comment:
`TransformUtil` seems to be an extensible class. We should provide API documentation in the header so downstream projects implementing the APIs have fully documented expected behavior. In general, headers are for API inheritance documentation, object files are for implementation detail documentation.

Reply:
Done